apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -165,12 +166,25 @@ export const saveModelCard = createAsyncThunk(
       currentAiCustomizations,
       savedAiCustomizations,
       EVENTS.SAVE_MODEL_CARD_INFO,
-      levelId,
+      levelPath,
       dispatch
     );
   }
 );
 
+const getLevelPath = (progress: ProgressState) => {
```

Comment:
Is there a reason not to just grab the URL (something like `window.location.pathname`)?

Reply:
Oh cool! I don't think so. I'll go ahead and use `window.location.pathname` instead. Thanks!

Reply:
FYI - when I added level url to Dance AI analytics, I added `currentScriptLevelUrl`.  https://github.com/code-dot-org/code-dot-org/pull/55531
But now I see I could've used `window.location.pathname` instead. Wonder if I should remove this seemingly unnecessary property.

Reply:
Yeah that seems redundant to me, but I could be missing something :)